docs/manuals/userman/sdm/scenario/index.md

```diff
@@ -288,6 +289,8 @@ returns the list of all existing scenarios. Here is the list of the parameters:
     order for dates, in alphabetical order for name and id, and in lexicographical order for tags.
     The default value is "name". If an incorrect sorting key is provided, the scenarios are sorted
     by name.
+- *created_start_time* represents the optional inclusive start date to filter scenarios by creation date.
+- *created_end_time* represents the optional inclusive end date to filter scenarios by creation date.
```

Comment:
Hum.

Usually, working on intervals, we include the start and **exclude** the end: `[start; end[`.
This is more convenient when it comes to use the API iteratively in a loop. So an iteration and the following are disjoint.

In other words, the two following calls can be safely treated independently.
```
taipy.get_scenarios(start, intermediate)
taipy.get_scenarios(intermediate, end)
```

Reply:
I understand. However, this would require similar changes on taipy. Should I handle that as well?

Reply:
Yep, please. Sorry about that. I missed it during the code review.